locales/english.json

```diff
@@ -4,9 +4,10 @@
   "common.documentTitle": "Data Marketplace",
   "auth.loading": "Signing you in\u2026",
   "auth.prompt": "Please sign in",
-  "frontpage.hero.title": "Find and get access to the data you need, when you need it",
-  "frontpage.hero.ingress": "Data Marketplace makes it easy to search for and request access to the data you need to do your job. Venenatis ipsum aliquet nibh at. Risus sapien erat vitae amet leo lobortis maecenas.",
-  "frontpage.popularProductsHeader": "Popular",
+  "frontpage.hero.title": "Find the data you need",
+  "frontpage.hero.ingress": "Equinor Data Marketplace makes it easy to find and request access to the data you need.",
```

Comment:
didn't we agree to only call it "Data Marketplace"? 🤔

Reply:
here's another suggestion to solve the orphan:
> Data Marketplace makes it easy for you to find and request access to the data you need, across Equinor.

Reply:
Let's add that text, liked it :)